.github/workflows/unit_tests_with_latest_deps.yml

```diff
@@ -37,36 +42,65 @@ jobs:
       - name: Install woodwork - tests requirements
         run: |
           python -m pip install unpacked_sdist/[test]
-      - if: ${{ matrix.libraries == 'spark' }}
-        name: Install woodwork - spark requirements
+      - name: Install Dask and Spark
         run: |
           sudo apt update
           sudo apt install -y openjdk-11-jre-headless
           JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"
           python -m pip install unpacked_sdist/[spark]
-      - if: ${{ matrix.libraries == 'dask' }}
-        name: Install woodwork - dask requirements
-        run: |
           python -m pip install unpacked_sdist/[dask]
-      - if: ${{ matrix.python_version != 3.8 }}
-        name: Run unit tests (no code coverage)
+      - if: ${{ matrix.python_version != 3.8 && matrix.directories == 'Testing to Disk with LatLong' }}
+        name: Run testing to Disk with LatLong Unit Tests (no code coverage)
         run: |
           cd unpacked_sdist
-          pytest woodwork/ -n 2
-        env:
-          PYARROW_IGNORE_TIMEZONE: 1
-          ALTERYX_OPEN_SRC_UPDATE_CHECKER: False
-      - if: ${{ matrix.python_version == 3.8 }}
-        name: Run unit tests with code coverage
+          pytest woodwork/tests/accessor/test_serialization.py::test_to_disk_with_latlong -n 2 --durations 0
+      - if: ${{ matrix.python_version != 3.8 && matrix.directories == 'All other Serialization' }}
+        name: Run all other Serialization Unit Tests (no code coverage)
+        run: |
+          cd unpacked_sdist
```

Comment:
Can we run `cd unpacked_sdist` before we get to all of these `if` statements? Would lead to marginal code cleanup, but would be nice to not have to repeat this call in every `if` cell